src/application-events-model.ts

```diff
@@ -587,7 +587,7 @@ export const ApplicationEventsModel: { [playerEventName: string]: KavaEvent } =
       eventType: ApplicationEventType.BUTTON_CLICKED,
       eventVar1: 'Advanced_captions_font_opacity',
       eventVar2: ButtonType.Choose,
-      eventVar3: payload
+      eventVar3: `${String(payload).split(".")[1]}%`
```

Comment:
does it possible if payload will be integer, like `10` (without decimal values)?